src/assets/sass/protocol/base/elements/_document.scss

```diff
@@ -7,18 +7,23 @@
 
 html {
     font-size: 100%;
-    background: #fff;
+    background: $color-white;
 }
 
 body {
     @include text-body-md;
-    background: #fff;
-    color: $color-black;
-    font-family: $font-base-family;
+    background: get-theme('background-color');
+    color: get-theme('body-text-color');
+    font-family: get-theme('body-font-family');
+    line-height: type-scale('body-line-height');
     -moz-osx-font-smoothing: grayscale;
     -webkit-font-smoothing: antialiased;
 }
 
+.mzp-t-dark {
```

Comment:
Declaring a foreground colour without a background colour has always made me uncomfortable. Last time we talked about it we agreed to remove it: https://github.com/mozilla/protocol/issues/515

If you're going to add the text colour change here it needs a background color declaration as well, and then we need to deal with the fallout from that.

I think this is a good time to do that actually since this is a breaking change.

If you'd like I can put together a separate PR with the mzp-t-dark changes to go live with the theme changes.

Reply:
I'm removing this. We don't really have a proper global dark theme, it's just been done on a per-component basis so I think adding such a sweeping rule here does more harm than good. And once I remove this I'll double-check all the components with dark versions.